scripts/format.py

```diff
@@ -20,9 +20,6 @@
     exit(-1)
 
 ver = subprocess.check_output(('clang-format', '--version'), text=True)
-if '10.' not in ver:
```

Comment:
I'd prefer we keep this code and update it to point to version 11 instead, if you can't install clang-format 11

Obviously that would require that we update the Code Format check run

Reply:
Sure! I had a go at changing it myself, but not sure if I did it correctly.